src/sentry/integrations/github/repository.py

```diff
@@ -112,13 +109,8 @@ def _format_commits(self, client, repo_name, commit_list):
     def _get_patchset(self, client, repo_name, sha):
         """Get the modified files for a commit
         """
-        key = u"get_commit:{}:{}".format(md5_text(repo_name).hexdigest(), sha)
```

Comment:
`get_commit` at the API client level is now being cached so caching is not needed here